persistence-jdbc/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/jdbc/JdbcPersistentEntityRegistry.scala

```diff
@@ -33,7 +33,7 @@ private[lagom] final class JdbcPersistentEntityRegistry @Inject() (system: Actor
 
   override protected def mapStartingOffset(storedOffset: Offset): AkkaOffset = storedOffset match {
     case Offset.NONE          => NoOffset
-    case seq: Offset.Sequence => Sequence(seq.value() + 1)
+    case seq: Offset.Sequence => Sequence(seq.value())
```

Comment:
Removes compensation for wrong offset query in  jdbc plugin. That's now fixed

Reply:
I think the whole reason that `mapStartingOffset` exists was to give us a place to put this compensation. Now that we don't need to compensate, we can get rid of the method and just inline the default implementation (which is `OffsetAdapter.dslOffsetToOffset(storedOffset)`).

Reply:
next time I will read the code!